lang/en.js

```diff
@@ -140,6 +140,10 @@ var theUILang =
  time_m				: "m ", /*for x minutes remaining*/
  time_s				: "s ", /*for x seconds remaining*/
  Base_directory			: "Directory",
+ Torrent_options		: "Torrent Options",
+ Return_select_label		: "Select",
+ Add_from_file			: "Add From File",
+ Add_from_URL				: "Add From URL",
```

Comment:
Could you resolve the spacing issues with `Add_from_URL` in the language files?

Reply:
Yeah, sure. They appeared to be in a line in VSCode, but jumped to right in GitHub. That's confusing.

Reply:
All you need to do is backspace one of the tabs.